vimiv/gui/metadatawidget.py

```diff
@@ -121,7 +121,12 @@ def metadata_list_keys(self, n_cols: int = 3, to_term: bool = False):
                 * ``--to-term``: Print the keys to the terminal instead.
             """
 
-            keys = sorted(set(self.handler.get_keys()))
+            try:
+                keys = sorted(set(self.handler.get_keys()))
+                _logger.debug("Successfully got keys")
+            except metadata.UnsupportedMetadataOperation:
```

Comment:
I would just not catch the exception here. The code would break anyway if the keys are not defined, but this should never happen as you say.

Reply:
Fixed in 16ec200